concent_api/common/decorators.py

```diff
@@ -298,3 +308,39 @@ def wrapper(*args, **kwargs):
             )
             raise
     return wrapper
+
+
+def get_response_from_view_with_db_retry(
+    view,
+    error: DatabaseError,
+    request: WSGIRequest,
+    client_message: golem_messages.message,
+    client_public_key: bytes,
+    *args: Optional[Any],
+    **kwargs: Optional[Any],
+) -> HttpResponse:
+    """
+    This section is to catch Database errors caused by multiple requests about one Subtask.
+    These requests are waiting and retrying.
+    """
+    log_string_message(
+        logger,
+        f'DatabaseError during process client request. Client public key: {client_public_key}.'
+        f'Error: {error}'
+    )
+    sleep_time = settings.INITIAL_VIEW_RETRY_DELAY
+
+    for counter in range(settings.MAX_VIEW_RETRIES):
+        try:
+            time.sleep(sleep_time)
+            sleep_time *= 2
```

Comment:
Do we really want to wait over 30 minutes for response ? I think the maximum is way to much and I would suggest shorter `INITIAL_VIEW_RETRY_DELAY`.

Reply:
Current settings are:
`MAX_VIEW_RETRIES = 10`
`INITIAL_VIEW_RETRY_DELAY = 1  # Value in seconds`
I think it was about 8 minutes.. But you are right. Is it ok??:
`MAX_VIEW_RETRIES = 10`
`INITIAL_VIEW_RETRY_DELAY = 0.5`

Reply:
The maximum delay time you can reach is `2**9=512` but you have already waited `256`, `128` etc. I think you should choose values that would overall not be longer that 1 minute. But please verify with @cameel too. I am thinking about it more like a response for web page, maybe in Golem network this can take much longer.